README.rst

```diff
@@ -146,20 +143,21 @@ Installing prerequisites on OpenBSD::
 
 To install the Synapse homeserver run::
 
-    virtualenv -p python2.7 ~/.synapse
-    source ~/.synapse/bin/activate
+    mkdir -p ~/synapse
+    virtualenv -p python3 ~/synapse/env
```

Comment:
Does python3 recommend using the venv module?

Reply:
I'm unfamiliar with the venv module.

Reply:
Its a standard library equivalent to virtualenv, but probably fine to stick with `virtualenv` for now